src/components/Chat/MembersSider/index.js

```diff
@@ -24,6 +25,11 @@ const MembersSider = props => {
     return classNameList.join(' ')
   }
 
+  const getIconMemberStatus = userId => {
+    const statusIcon = `chat-header-${getStatusById(userId)}-status-icon`
+    return statusIcon
+  }
+  
```

Comment:
Prettier + eslint plugins and autoformatting make life so much easier, if it's possible to use those.